### DISCUSSION TITLE
Revisiting Moodist’s Core Focus—Share Your Thoughts!

### DISCUSSION THREAD

**Discussion by remvze** [remvze/moodist#40] [General]
Hello everyone!

First off, thank you for the tremendous support Moodist has received!

As you know, over time, I added various productivity tools to Moodist, like a notepad, Pomodoro timer, and breathing exercises. However, I've come to realize that these additions have taken away from the project's original focus: simplicity. Consequently, Moodist has become a bit cluttered.

To realign with the core purpose of Moodist, I've decided to remove these extra tools and concentrate solely on ambient sounds. Before I proceed, I’d really appreciate your feedback.

What do you think?

Thanks again!

**Comment by Famku** [comment]
I totally agree

**Comment by arapl3y** [comment]
Sounds good to me.

**Comment by zmweske** [comment]
I don't think that most of them really detract, especially because they are very out of the way currently. I think the timer and breathing exercises pair really well with the app and are good being kept where they are (available but out of the way in a menu)

**Comment by CanaryFromTrantor** [comment]
necroing this, but as a new Moodist user I don't really mind them or notice them.  They seem to be nice to have and are hidden by default so the integration isn't obnoxious.  As others said earlier, they are out of they way.  Just can't thank you enough for the app.
